pkg/chartutil/requirements_test.go

```diff
@@ -25,22 +27,22 @@ import (
 )
 
 func TestLoadRequirements(t *testing.T) {
-	c, err := Load("testdata/frobnitz")
+	c, err := Load(filepath.Join("testdata", "frobnitz"))
```

Comment:
Another option would be adding a `filepath.FromSlash(...)' inside `Load()` to convert to os separators

Reply:
Yeah that'd work! Both approaches are good, though your suggestion should work fine and reduces LOC changed. 👍

Reply:
fixed.